device-catalog.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="ru">
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheey" href="device-catalog.css">
+        <title>Интернет магазин Device. Каталог товаров</title>
+    </head>
+    <body>
+        <header>
+            <div>
+                <a class="nav_link" href=".//index.html"><img src=".//img/logo-black.png" alt="Logo device"></a>
+                <div>
+                    <img src=".//img/search.svg">
+                    <input type="text" name="search" placeholder="Поиск по сайту">
+                    <input type="button" value="Найти">
+                </div>
+                <ul>
+                    <li><a href="#">username</a></li> 
+                    <!-- Тут должна быть какая-то переменная для username -->
+                    <li><a href="#">Выйти</a></li>
+                    <li><a href="#">Сравнить</a></li>
+                    <li><a href="#">Корзина</a></li>
+                </ul>                
+            </div>
+            <nav class="nav">                                                
+                <ul class="nav_list">
+                    <li class="#">
+                        <div class="catalog_div">
+                        <a href="#">Каталог товаров</a>
+                        <img src=".//img/plus.svg">
```

Comment:
it can be a pseudo element
always remember that images block page's rendering, the less images,the faster your site

Reply:
this mess is natural for beginners)